odoo/addons/base/migrations/10.0.1.3/end-migration.py

```diff
@@ -25,7 +26,26 @@ def convert_binary_attachment_style(env):
         openupgrade_90.convert_binary_field_to_attachment(env, field_spec)
 
 
+def merge_country_states(env):
+    group_list = [
+        'country_id', 'code',
```

Comment:
I would deduplicate by code only.

Reply:
Why?

Reply:
There are codes that are used by multiple countries. We cannot do that

Reply:
OK, I have mixed countries with states, so I was talking all the time about countries. Clear now.